### PR TITLE
feat(api): add simbridge integration

### PR DESCRIPTION
**Pull request by Lucky38i**
## Summary of Changes

This PR aims to integrate the fetch and packaging of the local-api into the a32nx addon.
* hash checks against new versions of the local-api
* Removal of the mcdu-server 

## Additional context

Discord username (if different from GitHub): Lcuky38i

## Testing instructions

**NOTE: steps 1-5 won't be needed when using the installer**
1. open `local-server.exe`
2.  The console will print a list of printers
img
4. copy the name of the printer (for example: `HP ENVY 5000 series [03F136]`) that you use and set that in the properties file found `resources/properties.json` and enable the printer
img
6. restart the server
7. navigate to `localhost:3838/interfaces/mcdu`(or whichever port the server console says) and observe mcdu web client
8. Ensure the mcdu web client performs as it did previously

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by 2hwk — APPROVED** [review]

**Comment by 2hwk** [comment]
Please solve conflicts

File: src/instruments/src/EFB/Localization/en.json [hunk 1]
```diff
@@ -487,8 +487,8 @@
       "ConesEnabled": "Safety Cones",
       "DefaultBarometerUnit": "Default Barometer Unit",
       "DynamicRegistrationDecal": "Dynamic Registration Decal",
-      "EnabledMcduServerConnectionAutoDeactivatesAfter5MinutesIfNoSuccessfulConnection": "Enable MCDU Server Connection (Auto deactivates after 5 minutes if no successful connection)",
-      "ExternalMcduServerPort": "External MCDU Server Port",
+      "EnabledSimbridgeConnectionAutoDeactivatesAfter5MinutesIfNoSuccessfulConnection": "Enable Simbridge Connection (Auto deactivates after 5 minutes if no successful connection)",
```
Comment (2hwk) [inline src/instruments/src/EFB/Localization/en.json:490]: For these localisation strings, you may have to coordinate with the localisation team to have these fixed, or just revert to using the older keys, which may be easier given that localazy is not exactly great at handling changing keys

**Review by 2hwk — CHANGES_REQUESTED** [review]

**Review by 2hwk — APPROVED** [review]

File: src/instruments/src/EFB/Settings/Pages/SimOptionsPage.tsx [hunk 2]
```diff
@@ -68,34 +68,34 @@ export const SimOptionsPage = () => {
                         </SelectGroup>
                     </SettingItem>
 
-                    <SettingItem name={t('Settings.SimOptions.EnabledMcduServerConnectionAutoDeactivatesAfter5MinutesIfNoSuccessfulConnection')}>
+                    <SettingItem name={t('Settings.SimOptions.EnableSimbridge')}>
```
Comment (frankkopp) [inline src/instruments/src/EFB/Settings/Pages/SimOptionsPage.tsx:71]:
```suggestion
                    <SettingItem name={t('Settings.SimOptions.EnableSimBridge')}>
```

File: src/instruments/src/EFB/Settings/Pages/SimOptionsPage.tsx [hunk 3]
```diff
@@ -68,34 +68,34 @@ export const SimOptionsPage = () => {
...
                             </SelectItem>
                         </SelectGroup>
                         <div className="pt-2 text-center">
-                            {mcduServerEnabled === 'AUTO ON' ? t('Settings.SimOptions.Active') : t('Settings.SimOptions.Inactive')}
+                            {simbridgeEnabled === 'AUTO ON' ? t('Settings.SimOptions.Active') : t('Settings.SimOptions.Inactive')}
                         </div>
                     </SettingItem>
 
-                    <SettingItem name={t('Settings.SimOptions.ExternalMcduServerPort')}>
+                    <SettingItem name={t('Settings.SimOptions.SimbridgePort')}>
```
Comment (frankkopp) [inline src/instruments/src/EFB/Settings/Pages/SimOptionsPage.tsx:93]:
```suggestion
                    <SettingItem name={t('Settings.SimOptions.SimBridgePort')}>
```

**Review by frankkopp — CHANGES_REQUESTED** [review]

File: src/instruments/src/EFB/Settings/Pages/SimOptionsPage.tsx (hunk 2 above)
Comment (2hwk) [inline src/instruments/src/EFB/Settings/Pages/SimOptionsPage.tsx:71]: <SettingItem name={t('Settings.SimOptions.EnableSimbridge')}>
```suggestion
                    <SettingItem name={t('Settings.SimOptions.EnableSimBridge')}>
```

**Review by 2hwk — CHANGES_REQUESTED** [review]

**Review by 2hwk — APPROVED** [review]

File: .github/CHANGELOG.md [hunk 4]
```diff
@@ -312,6 +312,7 @@
 1. [ATSU] Add FANS-C simulation for CPDLC - @Sven [de en] - (Sven Czarnian)
 1. [ATSU] Add VDL3-simulation of datalink transport protocol - @Sven [de en] - (Sven Czarnian)
 1. [FMGC] Improve robustness of ILS auto-tuning - @tracernz (Mike)
+1. [ADDON] Simbridge Integration - @lucky38i (Alex)
```
Comment (BlueberryKing) [inline .github/CHANGELOG.md:315]: Can you put this under 0.9.0?

**Comment by 2hwk** [comment]
QA Report

Discord: 2Cas#1022
Object of testing: #6993
Tier of Testing: 1
Date: 30/08/2022

Testing Process: 
- [x] Start SimBridge
- [x] navigate to localhost:3838/interfaces/mcdu(or whichever port the server console says)
- [x]  observe mcdu web client Ensure the mcdu web client performs as it did previously
- [x] Disconnected from then reconnected SimBridge, still functional. 

Negatives: N/A

Testing Results: Passed

Conclusions: LGTM

**Pull request merged**